webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,11 +7,22 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <base-button
+        v-if="showDeleteImageButton"
+        class="delete-image"
+        icon="close"
+        size="small"
+        circle
+        danger
+        filled
+        @click.prevent="deleteImage"
+      />
       <hc-teaser-image
         :contribution="contribution"
-        @addTeaserImage="addTeaserImage"
         :class="{ '--blur-image': form.blurImage }"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T19:49:32Z" title="Friday, December 20th 2019, 8:49:32 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -198,6 +204,7 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      showDeleteButton: this.contribution ? this.contribution.image : false,
```


```suggestion
      showDeleteButton: !!this.contribution,
```